.github/workflows/algolia-upload-index.yml

```diff
@@ -3,6 +3,7 @@ name: Algolia Upload Index
 env:
   ALGOLIA_PROJECT: ${{ secrets.ALGOLIA_PROJECT }}
   AGOLIA_WRITE_KEY: ${{ secrets.AGOLIA_WRITE_KEY }}
+  ALGOLIA_INDEX: ${{ secrets.ALGOLIA_INDEX }}
```

Comment:
Read `ALGOLIA_INDEX` from secrets instead of hard-coding the value in the script.

Reply:
Superseded by #187